web/src/app/util/Markdown.js

```diff
@@ -2,14 +2,63 @@ import React from 'react'
 import ReactMarkdown from 'react-markdown'
 import { safeURL } from './safeURL'
 import gfm from 'remark-gfm'
+import { makeStyles } from '@material-ui/core'
+
+const useStyles = makeStyles({
+  markdown: {
+    '& h1, h2, h3, h4, h5, h6, p': {
+      marginTop: 0,
+      marginBottom: '0.5rem',
+    },
+    '& hr': {
+      margin: '1.5rem 0',
+    },
+    '& th': {
+      textAlign: 'left',
+      paddingRight: '8px',
+    },
+    '& td': {
+      paddingRight: '8px',
+    },
+    '& pre': {
+      display: 'block',
+      padding: '9.5px',
+      margin: '0 0 10px',
+      fontSize: '13px',
+      lineHeight: '1.42857143',
+      color: '#333',
+      wordBreak: 'break-all',
+      wordWrap: 'break-word',
+      backgroundColor: '#f5f5f5',
+      border: '1px solid #ccc',
+      borderRadius: '4px',
+    },
```

Comment:
Everything is rendering as expected now, though imho I think we should be using the defaults from this library instead of this here and just focus on getting the code block elements rendered with the proper colors. To be blunt, I'm not really in favor of maintaining things like `lineHeight: '1.42857143'` if we can avoid it

Reply:
Doing some functional testing to see the differences, outside of the code elements I'm only seeing the CSS here compact the lines together.

Reply:
Since this PR is for removing the dead CSS but also refactoring, I'd be fine with making those changes here. We should be able to do away with everything except the color bits for `pre`/`code` and be in a good state. I went ahead and double-checked what we'll need to keep (also adjusting one px value to rem). Will comment with the commit suggestion

Reply:
I did some further digging, and [this code](https://git.target.com/goalert/goalert-ui/commit/5b304fdc0da5191306a66538a785357725f45b48) is a remnant from when we were using a completely different markdown library altogether (way back at the beginning of 2018)- so I definitely think we're good here to remove